includes/Modules/Analytics.php

```diff
@@ -1350,4 +1312,19 @@ protected function has_data_for_url( $url ) {
 
 		return (bool) $has_data;
 	}
+
+	/**
+	 * Determines the Analytics account ID from a given Analytics property ID.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $property_id Analytics property ID.
+	 * @return string Analytics account ID, or empty string if invalid property ID.
+	 */
+	protected function determine_account_id( $property_id ) {
```

Comment:
I would suggest we rename this to something like `parse_account_id` since it isn't doing anything more than that to determine it (such as make additional requests, etc). I suppose it _could_ but given a property ID, that would never be necessary 😄